xml/Microsoft.Extensions.DependencyInjection/HttpClientBuilderExtensions.xml

```diff
@@ -1,1033 +1,1060 @@
+      <MemberType>Method</MemberType>
+      <AssemblyInfo>
+        <AssemblyName>Microsoft.Extensions.Http</AssemblyName>
+        <AssemblyVersion>8.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>Microsoft.Extensions.DependencyInjection.IHttpClientBuilder</ReturnType>
+      </ReturnValue>
+      <Parameters>
+        <Parameter Name="builder" Type="Microsoft.Extensions.DependencyInjection.IHttpClientBuilder" Index="0" FrameworkAlternate="dotnet-plat-ext-8.0" RefType="this" />
+        <Parameter Name="httpClientLoggerFactory" Type="System.Func&lt;System.IServiceProvider,Microsoft.Extensions.Http.Logging.IHttpClientLogger&gt;" Index="1" FrameworkAlternate="dotnet-plat-ext-8.0" />
+        <Parameter Name="wrapHandlersPipeline" Type="System.Boolean" Index="2" FrameworkAlternate="dotnet-plat-ext-8.0" />
+      </Parameters>
+      <Docs>
+        <param name="builder">The <see cref="T:Microsoft.Extensions.DependencyInjection.IHttpClientBuilder" />.</param>
+        <param name="httpClientLoggerFactory">A delegate that is used to create a custom logger. The logger should implement <see cref="T:Microsoft.Extensions.Http.Logging.IHttpClientLogger" /> or <see cref="T:Microsoft.Extensions.Http.Logging.IHttpClientAsyncLogger" />.</param>
+        <param name="wrapHandlersPipeline">Whether the logging handler with the custom logger would be added to the top or to the bottom of the additional handlers chains.</param>
+        <summary>Adds a delegate that will be used to create an additional logger for a named <see cref="T:System.Net.Http.HttpClient" />. The custom logger would be invoked from a dedicated logging DelegatingHandler on every request of the corresponding named <see cref="T:System.Net.Http.HttpClient" />.</summary>
+        <returns>An <see cref="T:Microsoft.Extensions.DependencyInjection.IHttpClientBuilder" /> that can be used to configure the client.</returns>
+        <remarks>
+          <para> If the <see paramref="wrapHandlersPipeline" /> is `true`, <see cref="M:Microsoft.Extensions.Http.Logging.IHttpClientLogger.LogRequestStart(System.Net.Http.HttpRequestMessage)" /> and <see cref="M:Microsoft.Extensions.Http.Logging.IHttpClientAsyncLogger.LogRequestStartAsync(System.Net.Http.HttpRequestMessage,System.Threading.CancellationToken)" /> would be executed before all other additional handlers in the chain. <see cref="M:Microsoft.Extensions.Http.Logging.IHttpClientLogger.LogRequestStop(System.Object,System.Net.Http.HttpRequestMessage,System.Net.Http.HttpResponseMessage,System.TimeSpan)" /> and <see cref="M:Microsoft.Extensions.Http.Logging.IHttpClientAsyncLogger.LogRequestStopAsync(System.Object,System.Net.Http.HttpRequestMessage,System.Net.Http.HttpResponseMessage,System.TimeSpan,System.Threading.CancellationToken)" /> would be executed after all other additional handlers, essentially wrapping the whole pipeline.

```

Comment:
`paramref` doesn't seem to work 🥲 which is strange, because I copied it from somewhere where it was working... Does anyone know what is a proper way to reference a parameter?

Reply:
To reference a parameter, you need to use `<paramref name="parameterName" />`

Reply:
```suggestion
          <para> If the <paramref name="wrapHandlersPipeline" /> is `true`, <see cref="M:Microsoft.Extensions.Http.Logging.IHttpClientLogger.LogRequestStart(System.Net.Http.HttpRequestMessage)" /> and <see cref="M:Microsoft.Extensions.Http.Logging.IHttpClientAsyncLogger.LogRequestStartAsync(System.Net.Http.HttpRequestMessage,System.Threading.CancellationToken)" /> would be executed before all other additional handlers in the chain. <see cref="M:Microsoft.Extensions.Http.Logging.IHttpClientLogger.LogRequestStop(System.Object,System.Net.Http.HttpRequestMessage,System.Net.Http.HttpResponseMessage,System.TimeSpan)" /> and <see cref="M:Microsoft.Extensions.Http.Logging.IHttpClientAsyncLogger.LogRequestStopAsync(System.Object,System.Net.Http.HttpRequestMessage,System.Net.Http.HttpResponseMessage,System.TimeSpan,System.Threading.CancellationToken)" /> would be executed after all other additional handlers, essentially wrapping the whole pipeline.

```

Reply:
If there are more similar `<see paramref` to fix, can you please provide the suggestions in comments so I can apply them?

Reply:
I think I got all of them now, thanks!

I think it was my bad, and in this file I mixed up `<see cref=` and `<paramref name=`, because in other files I seem to have used correct tag 🤦‍♀️

btw in preview `false` and `true` also weren't formatted as expected, is there something else I should have used?

Reply:
No worries!

For reserved keywords, use `see langword`. Examples: `<see langword="false" />`, `<see langword="true" />` `<see langword="null" />`.